examples/guide/frontend/guide-with-step-navs.json

```diff
@@ -886,7 +886,8 @@
+        "hide_chapter_navigation": true
```

Comment:
@andysellick The tests are failing because this value is now in the wrong part of the guide 😞

Reply:
D'oh.